
Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-tgw5r[0m
{"level":"warn","ts":1585131760.4784534,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1585131760.4790606,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-pq92v[0m
{"level":"warn","ts":1585131761.9299817,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131761.9323792,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-mhq9j[0m
{"level":"warn","ts":1585131769.0848458,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131769.0872376,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1573-in-b7kdz[0m
{"level":"warn","ts":1585131769.4745722,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1585131773.1208622,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:7da6ec5f05371c738477fbb48333006227e0aec0,1573:a51f0bece63aa4d9455fafa802120b4de14ad020
DEBUG: ran git fetch --unshallow origin a51f0bece63aa4d9455fafa802120b4de14ad020: 7da6ec5f05371c738477fbb48333006227e0aec0: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 7da6ec5f05371c738477fbb48333006227e0aec0 in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge a51f0bece63aa4d9455fafa802120b4de14ad020 in 
Merged SHA a51f0bece63aa4d9455fafa802120b4de14ad020 with commit message 'Merge branch 'master' into 1571_deployment_name_changes' into base branch master
Merged SHA d01967f131d8b016cd67e73f9786fcf3e281444e with commit message 'review changes' into base branch master
Merged SHA 10d10e9901c39c4209ab6d7bb77b6d5562132a9d with commit message 'Merge branch 'master' into 1571_deployment_name_changes' into base branch master
Merged SHA ea47241c5c364b0f1d21b58efb0a12c16e8bfdc7 with commit message 'change deployment name convention' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1573
+ BUILD_NUMBER=7
+ PIPELINE_CONTEXT=integration
++ cat
+++ date
+ COMMENT_MSG='Wed Mar 25 10:22:59 UTC 2020
The logs for [integration] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1573/7.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1573 --build=7'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1573 '--comment=Wed Mar 25 10:22:59 UTC 2020
The logs for [integration] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1573/7.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1573 --build=7'

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1573-in-ftjjw[0m
{"level":"warn","ts":1585131781.5736861,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131781.5784683,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1573-in-kcrrb[0m
{"level":"warn","ts":1585131781.9842248,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131782.395601,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-credential-initializer-nbf79[0m
{"level":"warn","ts":1585131802.368916,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1585131802.3694534,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-working-dir-initializer-cvbrw[0m
{"level":"warn","ts":1585131803.7775826,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131803.7798684,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-create-dir-workspace-k9z5j[0m
{"level":"warn","ts":1585131810.9371102,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131810.9407346,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-source-copy-workspace-fkkwc[0m
{"level":"warn","ts":1585131811.285839,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1585131813.827812,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1573-in-nhqhm-7[0m stage [32mend-to-end[0m and container [32mstep-test-end-to-end[0m
Test run type is: [base]
Starting Docker: docker.
[SETUP] Waiting for Docker to be ready, sleeping for 1 seconds ...
kind create cluster --config ../resources/kind_config.yaml
Creating cluster "kind" ...
 ‚Ä¢ Ensuring node image (kindest/node:v1.16.3) üñº  ...
 ‚úì Ensuring node image (kindest/node:v1.16.3) üñº
 ‚Ä¢ Preparing nodes üì¶  ...
 ‚úì Preparing nodes üì¶
 ‚Ä¢ Writing configuration üìú  ...
 ‚úì Writing configuration üìú
 ‚Ä¢ Starting control-plane üïπÔ∏è  ...
 ‚úì Starting control-plane üïπÔ∏è
 ‚Ä¢ Installing CNI üîå  ...
 ‚úì Installing CNI üîå
 ‚Ä¢ Installing StorageClass üíæ  ...
 ‚úì Installing StorageClass üíæ
 ‚Ä¢ Joining worker nodes üöú  ...
 ‚úì Joining worker nodes üöú
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community üôÇ
`kind get kubeconfig-path` is deprecated!

KIND will export and merge kubeconfig like kops, minikube, etc.
This command is now unnecessary and will be removed in a future release.

For more info see: https://github.com/kubernetes-sigs/kind/issues/1060
See also the output of `kind create cluster`

Files changed in python folder:
SKIPPING PYTHON IMAGE BUILD...
Files changed in operator folder:
operator/apis/machinelearning/v1/seldondeployment_types.go
operator/apis/machinelearning/v1/seldondeployment_types_test.go
operator/constants/constants.go
operator/controllers/seldondeployment_controller.go
operator/controllers/seldondeployment_controller_test.go
cd ../../operator && make kind-image-install
make[1]: Entering directory '/workspace/source/operator'
go: creating new go.mod: module tmp
go: finding sigs.k8s.io/controller-tools v0.2.5
go: finding sigs.k8s.io/controller-tools/cmd v0.2.5
go: finding sigs.k8s.io v0.2.5
go: finding sigs.k8s.io/controller-tools/cmd/controller-gen v0.2.5
go: downloading sigs.k8s.io/controller-tools v0.2.5
go: extracting sigs.k8s.io/controller-tools v0.2.5
go: downloading gopkg.in/yaml.v3 v3.0.0-20190905181640-827449938966
go: downloading github.com/spf13/cobra v0.0.5
go: downloading sigs.k8s.io/yaml v1.1.0
go: downloading k8s.io/apimachinery v0.17.0
go: downloading golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/gobuffalo/flect v0.2.0
go: downloading k8s.io/apiextensions-apiserver v0.17.0
go: extracting sigs.k8s.io/yaml v1.1.0
go: extracting github.com/fatih/color v1.7.0
go: downloading k8s.io/api v0.17.0
go: extracting gopkg.in/yaml.v3 v3.0.0-20190905181640-827449938966
go: extracting github.com/gobuffalo/flect v0.2.0
go: downloading gopkg.in/yaml.v2 v2.2.4
go: extracting github.com/spf13/cobra v0.0.5
go: downloading github.com/mattn/go-colorable v0.1.2
go: downloading github.com/mattn/go-isatty v0.0.8
go: extracting github.com/mattn/go-colorable v0.1.2
go: extracting gopkg.in/yaml.v2 v2.2.4
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: extracting github.com/mattn/go-isatty v0.0.8
go: downloading github.com/spf13/pflag v1.0.5
go: extracting github.com/inconshreveable/mousetrap v1.0.0
go: downloading golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456
go: extracting k8s.io/apimachinery v0.17.0
go: extracting github.com/spf13/pflag v1.0.5
go: extracting k8s.io/apiextensions-apiserver v0.17.0
go: downloading github.com/google/gofuzz v1.0.0
go: downloading github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
go: downloading k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
go: downloading gopkg.in/inf.v0 v0.9.1
go: extracting github.com/google/gofuzz v1.0.0
go: downloading k8s.io/klog v1.0.0
go: downloading golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: extracting gopkg.in/inf.v0 v0.9.1
go: extracting k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
go: extracting k8s.io/klog v1.0.0
go: extracting golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
go: extracting golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456
go: extracting k8s.io/api v0.17.0
go: extracting golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: extracting github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
go: downloading golang.org/x/text v0.3.2
go: extracting golang.org/x/text v0.3.2
go: finding k8s.io/api v0.17.0
go: finding github.com/spf13/cobra v0.0.5
go: finding golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
go: finding github.com/fatih/color v1.7.0
go: finding k8s.io/apimachinery v0.17.0
go: finding gopkg.in/yaml.v3 v3.0.0-20190905181640-827449938966
go: finding k8s.io/apiextensions-apiserver v0.17.0
go: finding github.com/gobuffalo/flect v0.2.0
go: finding sigs.k8s.io/yaml v1.1.0
go: finding github.com/mattn/go-colorable v0.1.2
go: finding github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
go: finding github.com/mattn/go-isatty v0.0.8
go: finding github.com/spf13/pflag v1.0.5
go: finding github.com/google/gofuzz v1.0.0
go: finding gopkg.in/inf.v0 v0.9.1
go: finding gopkg.in/yaml.v2 v2.2.4
go: finding k8s.io/klog v1.0.0
go: finding k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
go: finding golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: finding golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456
go: finding golang.org/x/text v0.3.2
/builder/home/go/bin/controller-gen object:headerFile=./hack/boilerplate.go.txt paths="./..."
go fmt ./...
go vet ./...
go: downloading github.com/onsi/gomega v1.8.1
go: downloading github.com/onsi/ginkgo v1.11.0
go: downloading github.com/Azure/go-autorest/autorest v0.9.2
go: downloading github.com/gophercloud/gophercloud v0.4.0
go: extracting github.com/Azure/go-autorest/autorest v0.9.2
go: downloading github.com/Azure/go-autorest/autorest/adal v0.7.0
go: downloading github.com/Azure/go-autorest/logger v0.1.0
go: extracting github.com/onsi/gomega v1.8.1
go: extracting github.com/onsi/ginkgo v1.11.0
go: extracting github.com/Azure/go-autorest/autorest/adal v0.7.0
go: downloading github.com/Azure/go-autorest/autorest/date v0.2.0
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/Azure/go-autorest/tracing v0.5.0
go: downloading golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
go: extracting github.com/Azure/go-autorest/autorest/date v0.2.0
go: extracting github.com/Azure/go-autorest/logger v0.1.0
go: extracting github.com/Azure/go-autorest/tracing v0.5.0
go: downloading github.com/hpcloud/tail v1.0.0
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: extracting golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
go: extracting github.com/hpcloud/tail v1.0.0
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: extracting gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: extracting github.com/gophercloud/gophercloud v0.4.0
/builder/home/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
rm -rf generated
mkdir generated
kustomize build config/default/ -o generated
cp generated/apiextensions.k8s.io_v1beta1_customresourcedefinition_seldondeployments.machinelearning.seldon.io.yaml testing/machinelearning.seldon.io_seldondeployments.yaml
go test ./controllers/... ./apis/...  -coverprofile cover.out
ok  	github.com/seldonio/seldon-core/operator/controllers	19.810s	coverage: 49.0% of statements
?   	github.com/seldonio/seldon-core/operator/controllers/resources/credentials	[no test files]
ok  	github.com/seldonio/seldon-core/operator/controllers/resources/credentials/gcs	0.012s	coverage: 100.0% of statements
ok  	github.com/seldonio/seldon-core/operator/controllers/resources/credentials/s3	0.017s	coverage: 94.6% of statements
ok  	github.com/seldonio/seldon-core/operator/apis/machinelearning/v1	0.032s	coverage: 39.3% of statements
?   	github.com/seldonio/seldon-core/operator/apis/machinelearning/v1alpha2	[no test files]
?   	github.com/seldonio/seldon-core/operator/apis/machinelearning/v1alpha3	[no test files]
docker build . -t seldonio/seldon-core-operator:1.0.3-SNAPSHOT
Sending build context to Docker daemon  3.085MB
Step 1/17 : FROM golang:1.13.2 as builder
1.13.2: Pulling from library/golang
c7b7d16361e0: Pulling fs layer
b7a128769df1: Pulling fs layer
1128949d0793: Pulling fs layer
667692510b70: Pulling fs layer
c70d80036479: Pulling fs layer
a8669f8c1a42: Pulling fs layer
433331673060: Pulling fs layer
667692510b70: Waiting
c70d80036479: Waiting
a8669f8c1a42: Waiting
b7a128769df1: Verifying Checksum
b7a128769df1: Download complete
1128949d0793: Verifying Checksum
1128949d0793: Download complete
c7b7d16361e0: Verifying Checksum
c7b7d16361e0: Download complete
667692510b70: Verifying Checksum
667692510b70: Download complete
c70d80036479: Verifying Checksum
c70d80036479: Download complete
433331673060: Verifying Checksum
433331673060: Download complete
a8669f8c1a42: Verifying Checksum
a8669f8c1a42: Download complete
c7b7d16361e0: Pull complete
b7a128769df1: Pull complete
1128949d0793: Pull complete
667692510b70: Pull complete
c70d80036479: Pull complete
a8669f8c1a42: Pull complete
433331673060: Pull complete
Digest: sha256:b3086f21bfbe896973dd6147b46f2736a7deb2c04ed76219835befa0244a9562
Status: Downloaded newer image for golang:1.13.2
 ---> e37698ff7351
Step 2/17 : WORKDIR /workspace
 ---> Running in 2193fc2e7ee1
Removing intermediate container 2193fc2e7ee1
 ---> aee19d006b9d
Step 3/17 : COPY go.mod go.mod
 ---> a775629d5174
Step 4/17 : COPY go.sum go.sum
 ---> e83f8edb6bb2
Step 5/17 : RUN go mod download
 ---> Running in ccfaf1550c2d
[91mgo: finding cloud.google.com/go v0.38.0
[0m[91mgo: finding github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
[0m[91mgo: finding github.com/Azure/go-autorest/autorest v0.9.2
[0m[91mgo: finding github.com/Azure/go-autorest/autorest/adal v0.7.0
[0m[91mgo: finding github.com/Azure/go-autorest/autorest/date v0.2.0
[0m[91mgo: finding github.com/Azure/go-autorest/autorest/mocks v0.3.0
[0m[91mgo: finding github.com/Azure/go-autorest/logger v0.1.0
[0m[91mgo: finding github.com/Azure/go-autorest/tracing v0.5.0
[0m[91mgo: finding github.com/BurntSushi/toml v0.3.1
[0m[91mgo: finding github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802
[0m[91mgo: finding github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
[0m[91mgo: finding github.com/PuerkitoBio/purell v1.1.1
[0m[91mgo: finding github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
[0m[91mgo: finding github.com/agnivade/levenshtein v1.0.1
[0m[91mgo: finding github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc
[0m[91mgo: finding github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf
[0m[91mgo: finding github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
[0m[91mgo: finding github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6
[0m[91mgo: finding github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
[0m[91mgo: finding github.com/beorn7/perks v1.0.0
[0m[91mgo: finding github.com/bgentry/speakeasy v0.1.0
[0m[91mgo: finding github.com/blang/semver v3.5.0+incompatible
[0m[91mgo: finding github.com/client9/misspell v0.3.4
[0m[91mgo: finding github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa
[0m[91mgo: finding github.com/coreos/etcd v3.3.10+incompatible
[0m[91mgo: finding github.com/coreos/go-etcd v2.0.0+incompatible
[0m[91mgo: finding github.com/coreos/go-oidc v2.1.0+incompatible
[0m[91mgo: finding github.com/coreos/go-semver v0.3.0
[0m[91mgo: finding github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
[0m[91mgo: finding github.com/coreos/pkg v0.0.0-20180108230652-97fdf19511ea
[0m[91mgo: finding github.com/cpuguy83/go-md2man v1.0.10
[0m[91mgo: finding github.com/creack/pty v1.1.7
[0m[91mgo: finding github.com/davecgh/go-spew v1.1.1
[0m[91mgo: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
[0m[91mgo: finding github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
[0m[91mgo: finding github.com/docker/go-units v0.4.0
[0m[91mgo: finding github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
[0m[91mgo: finding github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815
[0m[91mgo: finding github.com/dustin/go-humanize v1.0.0
[0m[91mgo: finding github.com/elazarl/goproxy v0.0.0-20170405201442-c4fc26588b6e
[0m[91mgo: finding github.com/emicklei/go-restful v2.9.5+incompatible
[0m[91mgo: finding github.com/evanphx/json-patch v4.5.0+incompatible
[0m[91mgo: finding github.com/fatih/color v1.7.0
[0m[91mgo: finding github.com/fsnotify/fsnotify v1.4.7
[0m[91mgo: finding github.com/ghodss/yaml v1.0.0
[0m[91mgo: finding github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
[0m[91mgo: finding github.com/go-kit/kit v0.8.0
[0m[91mgo: finding github.com/go-logfmt/logfmt v0.3.0
[0m[91mgo: finding github.com/go-logr/logr v0.1.0
[0m[91mgo: finding github.com/go-logr/zapr v0.1.0
[0m[91mgo: finding github.com/go-openapi/analysis v0.19.5
[0m[91mgo: finding github.com/go-openapi/errors v0.19.2
[0m[91mgo: finding github.com/go-openapi/jsonpointer v0.19.3
[0m[91mgo: finding github.com/go-openapi/jsonreference v0.19.3
[0m[91mgo: finding github.com/go-openapi/loads v0.19.4
[0m[91mgo: finding github.com/go-openapi/runtime v0.19.4
[0m[91mgo: finding github.com/go-openapi/spec v0.19.3
[0m[91mgo: finding github.com/go-openapi/strfmt v0.19.3
[0m[91mgo: finding github.com/go-openapi/swag v0.19.5
[0m[91mgo: finding github.com/go-openapi/validate v0.19.5
[0m[91mgo: finding github.com/go-stack/stack v1.8.0
[0m[91mgo: finding github.com/gogo/protobuf v1.3.1
[0m[91mgo: finding github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
[0m[91mgo: finding github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
[0m[91mgo: finding github.com/golang/mock v1.2.0
[0m[91mgo: finding github.com/golang/protobuf v1.3.2
[0m[91mgo: finding github.com/google/btree v1.0.0
[0m[91mgo: finding github.com/google/go-cmp v0.3.1
[0m[91mgo: finding github.com/google/gofuzz v1.0.0
[0m[91mgo: finding github.com/google/martian v2.1.0+incompatible
[0m[91mgo: finding github.com/google/pprof v0.0.0-20181206194817-3ea8567a2e57
[0m[91mgo: finding github.com/google/uuid v1.1.1
[0m[91mgo: finding github.com/googleapis/gax-go/v2 v2.0.4
[0m[91mgo: finding github.com/googleapis/gnostic v0.3.1
[0m[91mgo: finding github.com/gophercloud/gophercloud v0.4.0
[0m[91mgo: finding github.com/gorilla/websocket v1.4.0
[0m[91mgo: finding github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7
[0m[91mgo: finding github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
[0m[91mgo: finding github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
[0m[91mgo: finding github.com/grpc-ecosystem/grpc-gateway v1.9.5
[0m[91mgo: finding github.com/hashicorp/golang-lru v0.5.4
[0m[91mgo: finding github.com/hashicorp/hcl v1.0.0
[0m[91mgo: finding github.com/hpcloud/tail v1.0.0
[0m[91mgo: finding github.com/imdario/mergo v0.3.7
[0m[91mgo: finding github.com/inconshreveable/mousetrap v1.0.0
[0m[91mgo: finding github.com/jonboulle/clockwork v0.1.0
[0m[91mgo: finding github.com/json-iterator/go v1.1.9
[0m[91mgo: finding github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024
[0m[91mgo: finding github.com/julienschmidt/httprouter v1.2.0
[0m[91mgo: finding github.com/kisielk/errcheck v1.2.0
[0m[91mgo: finding github.com/kisielk/gotool v1.0.0
[0m[91mgo: finding github.com/konsorten/go-windows-terminal-sequences v1.0.1
[0m[91mgo: finding github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
[0m[91mgo: finding github.com/kr/pretty v0.1.0
[0m[91mgo: finding github.com/kr/pty v1.1.5
[0m[91mgo: finding github.com/kr/text v0.1.0
[0m[91mgo: finding github.com/magiconair/properties v1.8.0
[0m[91mgo: finding github.com/mailru/easyjson v0.7.0
[0m[91mgo: finding github.com/mattn/go-colorable v0.0.9
[0m[91mgo: finding github.com/mattn/go-isatty v0.0.4
[0m[91mgo: finding github.com/mattn/go-runewidth v0.0.2
[0m[91mgo: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
[0m[91mgo: finding github.com/mitchellh/go-homedir v1.1.0
[0m[91mgo: finding github.com/mitchellh/mapstructure v1.1.2
[0m[91mgo: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
[0m[91mgo: finding github.com/modern-go/reflect2 v1.0.1
[0m[91mgo: finding github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
[0m[91mgo: finding github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223
[0m[91mgo: finding github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f
[0m[91mgo: finding github.com/olekukonko/tablewriter v0.0.0-20170122224234-a0225b3f23b5
[0m[91mgo: finding github.com/onsi/ginkgo v1.11.0
[0m[91mgo: finding github.com/onsi/gomega v1.8.1
[0m[91mgo: finding github.com/pborman/uuid v1.2.0
[0m[91mgo: finding github.com/pelletier/go-toml v1.2.0
[0m[91mgo: finding github.com/peterbourgon/diskv v2.0.1+incompatible
[0m[91mgo: finding github.com/pkg/errors v0.8.1
[0m[91mgo: finding github.com/pmezard/go-difflib v1.0.0
[0m[91mgo: finding github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021
[0m[91mgo: finding github.com/prometheus/client_golang v1.0.0
[0m[91mgo: finding github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
[0m[91mgo: finding github.com/prometheus/common v0.4.1
[0m[91mgo: finding github.com/prometheus/procfs v0.0.2
[0m[91mgo: finding github.com/remyoudompheng/bigfft v0.0.0-20170806203942-52369c62f446
[0m[91mgo: finding github.com/rogpeppe/fastuuid v0.0.0-20150106093220-6724a57986af
[0m[91mgo: finding github.com/russross/blackfriday v1.5.2
[0m[91mgo: finding github.com/sergi/go-diff v1.0.0
[0m[91mgo: finding github.com/sirupsen/logrus v1.4.2
[0m[91mgo: finding github.com/soheilhy/cmux v0.1.4
[0m[91mgo: finding github.com/spf13/afero v1.2.2
[0m[91mgo: finding github.com/spf13/cast v1.3.0
[0m[91mgo: finding github.com/spf13/cobra v0.0.5
[0m[91mgo: finding github.com/spf13/jwalterweatherman v1.0.0
[0m[91mgo: finding github.com/spf13/pflag v1.0.5
[0m[91mgo: finding github.com/spf13/viper v1.3.2
[0m[91mgo: finding github.com/stretchr/ob** v0.2.0
[0m[91mgo: finding github.com/stretchr/testify v1.4.0
[0m[91mgo: finding github.com/tidwall/pretty v1.0.0
[0m[91mgo: finding github.com/tmc/grpc-websocket-proxy v0.0.0-20170815181823-89b8d40f7ca8
[0m[91mgo: finding github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8
[0m[91mgo: finding github.com/urfave/cli v1.20.0
[0m[91mgo: finding github.com/vektah/gqlparser v1.1.2
[0m[91mgo: finding github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2
[0m[91mgo: finding github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77
[0m[91mgo: finding go.etcd.io/bbolt v1.3.3
[0m[91mgo: finding go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
[0m[91mgo: finding go.mongodb.org/mongo-driver v1.1.2
[0m[91mgo: finding go.opencensus.io v0.21.0
[0m[91mgo: finding go.uber.org/atomic v1.3.2
[0m[91mgo: finding go.uber.org/multierr v1.1.0
[0m[91mgo: finding go.uber.org/zap v1.10.0
[0m[91mgo: finding golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
[0m[91mgo: finding golang.org/x/exp v0.0.0-20190312203227-4b39c73a6495
[0m[91mgo: finding golang.org/x/image v0.0.0-20190227222117-0694c2d4d067
[0m[91mgo: finding golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3
[0m[91mgo: finding golang.org/x/mobile v0.0.0-20190312151609-d3739f865fa6
[0m[91mgo: finding golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
[0m[91mgo: finding golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
[0m[91mgo: finding golang.org/x/sync v0.0.0-20190423024810-112230192c58
[0m[91mgo: finding golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe
[0m[91mgo: finding golang.org/x/text v0.3.2
[0m[91mgo: finding golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0
[0m[91mgo: finding golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
[0m[91mgo: finding golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
[0m[91mgo: finding gomodules.xyz/jsonpatch/v2 v2.0.1
[0m[91mgo: finding gonum.org/v1/gonum v0.0.0-20190331200053-3d26580ed485
[0m[91mgo: finding gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e
[0m[91mgo: finding google.golang.org/api v0.4.0
[0m[91mgo: finding google.golang.org/appengine v1.6.5
[0m[91mgo: finding google.golang.org/genproto v0.0.0-20190916214212-f660b8655731
[0m[91mgo: finding google.golang.org/grpc v1.23.1
[0m[91mgo: finding gopkg.in/alecthomas/kingpin.v2 v2.2.6
[0m[91mgo: finding gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
[0m[91mgo: finding gopkg.in/cheggaaa/pb.v1 v1.0.25
[0m[91mgo: finding gopkg.in/fsnotify.v1 v1.4.7
[0m[91mgo: finding gopkg.in/inf.v0 v0.9.1
[0m[91mgo: finding gopkg.in/natefinch/lumberjack.v2 v2.0.0
[0m[91mgo: finding gopkg.in/resty.v1 v1.12.0
[0m[91mgo: finding gopkg.in/square/go-jose.v2 v2.2.2
[0m[91mgo: finding gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
[0m[91mgo: finding gopkg.in/yaml.v2 v2.2.4
[0m[91mgo: finding gotest.tools v2.2.0+incompatible
[0m[91mgo: finding honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc
[0m[91mgo: finding istio.io/api v0.0.0-20200305220354-742c3015842e
[0m[91mgo: finding istio.io/client-go v0.0.0-20200305195406-fddfc7a9ca06
[0m[91mgo: finding istio.io/gogo-genproto v0.0.0-20190930162913-45029607206a
[0m[91mgo: finding k8s.io/api v0.17.2
[0m[91mgo: finding k8s.io/apiextensions-apiserver v0.17.2
[0m[91mgo: finding k8s.io/apimachinery v0.17.2
[0m[91mgo: finding k8s.io/apiserver v0.17.2
[0m[91mgo: finding k8s.io/client-go v0.17.2
[0m[91mgo: finding k8s.io/code-generator v0.17.2
[0m[91mgo: finding k8s.io/component-base v0.17.2
[0m[91mgo: finding k8s.io/gengo v0.0.0-20190822140433-26a664648505
[0m[91mgo: finding k8s.io/klog v1.0.0
[0m[91mgo: finding k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
[0m[91mgo: finding k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
[0m[91mgo: finding knative.dev/pkg v0.0.0-20200306225627-d1665814487e
[0m[91mgo: finding modernc.org/cc v1.0.0
[0m[91mgo: finding modernc.org/golex v1.0.0
[0m[91mgo: finding modernc.org/mathutil v1.0.0
[0m[91mgo: finding modernc.org/strutil v1.0.0
[0m[91mgo: finding modernc.org/xc v1.0.0
[0m[91mgo: finding sigs.k8s.io/controller-runtime v0.5.0
[0m[91mgo: finding sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
[0m[91mgo: finding sigs.k8s.io/yaml v1.1.0
[0mRemoving intermediate container ccfaf1550c2d
 ---> e6fc96bde52e
Step 6/17 : COPY main.go main.go
 ---> 2bbb0005c065
Step 7/17 : COPY apis/ apis/
 ---> ede126c9b089
Step 8/17 : COPY controllers/ controllers/
 ---> c226c0e0d435
Step 9/17 : COPY utils/ utils/
 ---> 3415be140ec5
Step 10/17 : COPY constants/ constants/
 ---> 3fddf7945dc7
Step 11/17 : COPY client/ client/
 ---> c16465e412e1
Step 12/17 : RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 ---> Running in 0ed127dd4bd7
Removing intermediate container 0ed127dd4bd7
 ---> da6e45a441e0
Step 13/17 : FROM gcr.io/distroless/static:latest
latest: Pulling from distroless/static
9ff2acc3204b: Pulling fs layer
9ff2acc3204b: Verifying Checksum
9ff2acc3204b: Download complete
9ff2acc3204b: Pull complete
Digest: sha256:c6d5981545ce1406d33e61434c61e9452dad93ecd8397c41e89036ef977a88f4
Status: Downloaded newer image for gcr.io/distroless/static:latest
 ---> e4d2a899b1bf
Step 14/17 : WORKDIR /
 ---> Running in 68d4910f6967
Removing intermediate container 68d4910f6967
 ---> 4cc65562fc1f
Step 15/17 : COPY --from=builder /workspace/manager .
 ---> 61d43967d1e4
Step 16/17 : COPY licenses/license.txt .
 ---> 9256ddd80e7b
Step 17/17 : ENTRYPOINT ["/manager"]
 ---> Running in d5e01687176b
Removing intermediate container d5e01687176b
 ---> 81bd06afea3d
Successfully built 81bd06afea3d
Successfully tagged seldonio/seldon-core-operator:1.0.3-SNAPSHOT
docker save seldonio/seldon-core-operator:1.0.3-SNAPSHOT > operator.tar
kind load image-archive operator.tar 
make[1]: Leaving directory '/workspace/source/operator'
Files changed in engine folder:
SKIPPING ENGINE IMAGE BUILD...
Files changed in executor folder:
SKIPPING EXECUTOR IMAGE BUILD...
Build fixed models
cd ../docker/fixed-model && make kind_load_images
make[1]: Entering directory '/workspace/source/testing/docker/fixed-model'
s2i build -E environment_rest_v1 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.1
---> Installing application source...
Build completed successfully
s2i build -E environment_rest_v2 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.2
---> Installing application source...
Build completed successfully
kind load -v 3 docker-image seldonio/fixed-model:0.1
Image: "seldonio/fixed-model:0.1" with ID "sha256:98aed67378c265d00a7c7b2fee026e8517eefc69d2d2a0222214e0951c7e2b41" not present on node "kind-control-plane"
Image: "seldonio/fixed-model:0.1" with ID "sha256:98aed67378c265d00a7c7b2fee026e8517eefc69d2d2a0222214e0951c7e2b41" not present on node "kind-worker"
kind load -v 3 docker-image seldonio/fixed-model:0.2
Image: "seldonio/fixed-model:0.2" with ID "sha256:5dd30341a0d3d9da7832c44065ae01062d25534339270d24f278975bf8159d4f" not present on node "kind-control-plane"
Image: "seldonio/fixed-model:0.2" with ID "sha256:5dd30341a0d3d9da7832c44065ae01062d25534339270d24f278975bf8159d4f" not present on node "kind-worker"
make[1]: Leaving directory '/workspace/source/testing/docker/fixed-model'
kubectl create namespace seldon || echo "Namespace seldon already exists"
namespace/seldon created
kubectl create namespace test1 || echo "Namespace test1 already exists"
namespace/test1 created
helm repo add stable https://kubernetes-charts.storage.googleapis.com/
"stable" has been added to your repositories
helm repo add seldonio https://storage.googleapis.com/seldon-charts
"seldonio" has been added to your repositories
helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
"jaegertracing" has been added to your repositories
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "seldonio" chart repository
...Successfully got an update from the "jaegertracing" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ‚éà Happy Helming!‚éà 
helm install ambassador \
	stable/ambassador \
	-f ../resources/ambassador_values.yaml \
	--set crds.keep=false \
	--namespace seldon \
	--set replicaCount=1 \
	--wait
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
NAME: ambassador
LAST DEPLOYED: Wed Mar 25 10:33:24 2020
NAMESPACE: seldon
STATUS: deployed
REVISION: 1
NOTES:
Congratulations! You've successfully installed Ambassador.

For help, visit our Slack at https://d6e.co/slack or view the documentation online at https://www.getambassador.io.

To get the IP address of Ambassador, run the following commands:
  export NODE_PORT=$(kubectl get --namespace seldon -o jsonpath="{.spec.ports[0].nodePort}" services ambassador)
  export NODE_IP=$(kubectl get nodes --namespace seldon -o jsonpath="{.items[0].status.addresses[0].address}")
  echo http://$NODE_IP:$NODE_PORT
kubectl apply -f istio-1.4.2.yaml
clusterrole.rbac.authorization.k8s.io/istio-reader-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-reader-istio-system created
customresourcedefinition.apiextensions.k8s.io/attributemanifests.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/clusterrbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/destinationrules.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/envoyfilters.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/gateways.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/meshpolicies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/policies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/rbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/rules.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceentries.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/servicerolebindings.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceroles.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/virtualservices.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/adapters.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/instances.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/templates.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/handlers.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/sidecars.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/authorizationpolicies.security.istio.io created
namespace/istio-system created
serviceaccount/istio-reader-service-account created
clusterrole.rbac.authorization.k8s.io/istio-citadel-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-citadel-istio-system created
deployment.apps/istio-citadel created
poddisruptionbudget.policy/istio-citadel created
service/istio-citadel created
serviceaccount/istio-citadel-service-account created
deployment.apps/istio-egressgateway created
poddisruptionbudget.policy/istio-egressgateway created
gateway.networking.istio.io/istio-multicluster-egressgateway created
virtualservice.networking.istio.io/istio-multicluster-egressgateway created
envoyfilter.networking.istio.io/istio-multicluster-egressgateway created
destinationrule.networking.istio.io/istio-multicluster-destinationrule created
service/istio-egressgateway created
role.rbac.authorization.k8s.io/istio-egressgateway-sds created
rolebinding.rbac.authorization.k8s.io/istio-egressgateway-sds created
serviceaccount/istio-egressgateway-service-account created
clusterrole.rbac.authorization.k8s.io/istio-galley-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-galley-*****-role-binding-istio-system created
configmap/istio-mesh-galley created
configmap/istio-galley-configuration created
deployment.apps/istio-galley created
poddisruptionbudget.policy/istio-galley created
service/istio-galley created
serviceaccount/istio-galley-service-account created
configmap/istio-grafana-configuration-dashboards-citadel-dashboard created
configmap/istio-grafana-configuration-dashboards-galley-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-mesh-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-performance-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-service-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-workload-dashboard created
configmap/istio-grafana-configuration-dashboards-mixer-dashboard created
configmap/istio-grafana-configuration-dashboards-pilot-dashboard created
configmap/istio-grafana created
deployment.apps/grafana created
policy.authentication.istio.io/grafana-ports-mtls-disabled created
service/grafana created
deployment.apps/istio-ingressgateway created
gateway.networking.istio.io/ingressgateway created
poddisruptionbudget.policy/ingressgateway created
service/istio-ingressgateway created
serviceaccount/istio-ingressgateway-service-account created
sidecar.networking.istio.io/default created
clusterrole.rbac.authorization.k8s.io/istio-sidecar-injector-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-sidecar-injector-*****-role-binding-istio-system created
configmap/injector-mesh created
deployment.apps/istio-sidecar-injector created
mutatingwebhookconfiguration.admissionregistration.k8s.io/istio-sidecar-injector created
poddisruptionbudget.policy/istio-sidecar-injector created
service/istio-sidecar-injector created
serviceaccount/istio-sidecar-injector-service-account created
configmap/istio-sidecar-injector created
clusterrole.rbac.authorization.k8s.io/kiali created
clusterrole.rbac.authorization.k8s.io/kiali-viewer created
clusterrolebinding.rbac.authorization.k8s.io/kiali created
configmap/kiali created
secret/kiali created
deployment.apps/kiali created
service/kiali created
serviceaccount/kiali-service-account created
clusterrole.rbac.authorization.k8s.io/istio-pilot-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-istio-system created
configmap/pilot-envoy-config created
configmap/istio created
deployment.apps/istio-pilot created
meshpolicy.authentication.istio.io/default created
poddisruptionbudget.policy/istio-pilot created
service/istio-pilot created
serviceaccount/istio-pilot-service-account created
clusterrole.rbac.authorization.k8s.io/istio-policy created
clusterrolebinding.rbac.authorization.k8s.io/istio-policy-*****-role-binding-istio-system created
destinationrule.networking.istio.io/istio-policy created
deployment.apps/istio-policy created
poddisruptionbudget.policy/istio-policy created
service/istio-policy created
serviceaccount/istio-policy-service-account created
clusterrole.rbac.authorization.k8s.io/prometheus-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-istio-system created
configmap/prometheus created
deployment.apps/prometheus created
service/prometheus created
serviceaccount/prometheus created
horizontalpodautoscaler.autoscaling/istio-telemetry created
clusterrole.rbac.authorization.k8s.io/istio-mixer-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-mixer-*****-role-binding-istio-system created
attributemanifest.config.istio.io/istioproxy created
attributemanifest.config.istio.io/kubernetes created
handler.config.istio.io/stdio created
instance.config.istio.io/accesslog created
instance.config.istio.io/tcpaccesslog created
rule.config.istio.io/stdio created
rule.config.istio.io/stdiotcp created
instance.config.istio.io/requestcount created
instance.config.istio.io/requestduration created
instance.config.istio.io/requestsize created
instance.config.istio.io/responsesize created
instance.config.istio.io/tcpbytesent created
instance.config.istio.io/tcpbytereceived created
instance.config.istio.io/tcpconnectionsopened created
instance.config.istio.io/tcpconnectionsclosed created
handler.config.istio.io/prometheus created
rule.config.istio.io/promhttp created
rule.config.istio.io/promtcp created
rule.config.istio.io/promtcpconnectionopen created
rule.config.istio.io/promtcpconnectionclosed created
handler.config.istio.io/kubernetesenv created
rule.config.istio.io/kubeattrgenrulerule created
rule.config.istio.io/tcpkubeattrgenrulerule created
instance.config.istio.io/attributes created
destinationrule.networking.istio.io/istio-telemetry created
configmap/telemetry-envoy-config created
deployment.apps/istio-telemetry created
poddisruptionbudget.policy/istio-telemetry created
service/istio-telemetry created
serviceaccount/istio-mixer-service-account created
deployment.apps/istio-tracing created
service/jaeger-query created
service/jaeger-collector created
service/jaeger-agent created
service/zipkin created
service/tracing created
kubectl rollout status deployment.apps/istio-ingressgateway -n istio-system
Waiting for deployment "istio-ingressgateway" rollout to finish: 0 of 1 updated replicas are available...
deployment "istio-ingressgateway" successfully rolled out
kubectl rollout status deployment.apps/istio-pilot -n istio-system
deployment "istio-pilot" successfully rolled out
kubectl rollout status deployment.apps/istio-citadel -n istio-system
deployment "istio-citadel" successfully rolled out
helm install jaeger-operator \
	jaegertracing/jaeger-operator \
	--set rbac.clusterRole=true \
	--namespace seldon \
	--wait
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
NAME: jaeger-operator
LAST DEPLOYED: Wed Mar 25 10:35:51 2020
NAMESPACE: seldon
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
jaeger-operator is installed.


Check the jaeger-operator logs
  export POD=$(kubectl get pods-l app.kubernetes.io/instance=jaeger-operator -lapp.kubernetes.io/name=jaeger-operator --namespace seldon --output name)
  kubectl logs $POD --namespace=seldon
kubectl apply -f ../resources/jaeger.yaml --namespace seldon
jaeger.jaegertracing.io/jaeger created
mapping.getambassador.io/jaeger created
cd ../../operator && make install-cert-manager
make[1]: Entering directory '/workspace/source/operator'
kubectl create namespace cert-manager || echo "Namespace cert-manager-exists"
namespace/cert-manager created
kubectl label namespace cert-manager cert-manager.io/disable-validation=true || echo "namespace cert-manager-already labelled"
namespace/cert-manager labeled
kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
customresourcedefinition.apiextensions.k8s.io/certificaterequests.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/certificates.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/challenges.acme.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/clusterissuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/issuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/orders.acme.cert-manager.io created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
namespace/cert-manager configured
serviceaccount/cert-manager-cainjector created
serviceaccount/cert-manager created
serviceaccount/cert-manager-webhook created
clusterrole.rbac.authorization.k8s.io/cert-manager-cainjector created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-cainjector created
role.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-webhook:auth-delegator created
rolebinding.rbac.authorization.k8s.io/cert-manager-webhook:webhook-authentication-reader created
clusterrole.rbac.authorization.k8s.io/cert-manager-webhook:webhook-requester created
role.rbac.authorization.k8s.io/cert-manager:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager:leaderelection created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrole.rbac.authorization.k8s.io/cert-manager-view created
clusterrole.rbac.authorization.k8s.io/cert-manager-edit created
service/cert-manager created
service/cert-manager-webhook created
deployment.apps/cert-manager-cainjector created
deployment.apps/cert-manager created
deployment.apps/cert-manager-webhook created
mutatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
validatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
kubectl rollout status deployment.apps/cert-manager -n cert-manager
Waiting for deployment "cert-manager" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-cainjector -n cert-manager
Waiting for deployment "cert-manager-cainjector" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager-cainjector" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-webhook -n cert-manager
Waiting for deployment "cert-manager-webhook" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager-webhook" successfully rolled out
make[1]: Leaving directory '/workspace/source/operator'
sleep 5 #https://github.com/jetstack/cert-manager/issues/2273
kubectl create namespace seldon-system || echo "namespace seldon-system exists"
namespace/seldon-system created
helm install seldon \
	../../helm-charts/seldon-core-operator \
	--namespace seldon-system \
	--set istio.enabled=true \
	--set istio.gateway=seldon-gateway \
	--set certManager.enabled=false \
	--set executor.enabled=true \
	--wait
NAME: seldon
LAST DEPLOYED: Wed Mar 25 10:36:40 2020
NAMESPACE: seldon-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
kubectl config set-context $(kubectl config current-context) --namespace=seldon
Context "kind-kind" modified.
make: Entering directory '/workspace/source/python'
pip install -e . -r requirements-dev.txt
Obtaining file:///workspace/source/python
Collecting Flask<2.0.0
  Downloading Flask-1.1.1-py2.py3-none-any.whl (94 kB)
Collecting Flask-cors<4.0.0
  Downloading Flask_Cors-3.0.8-py2.py3-none-any.whl (14 kB)
Collecting redis<4.0.0
  Downloading redis-3.4.1-py2.py3-none-any.whl (71 kB)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 4)) (2.23.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 5)) (1.18.1)
Collecting flatbuffers<2.0.0
  Downloading flatbuffers-1.12-py2.py3-none-any.whl (15 kB)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 7)) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 8)) (1.27.2)
Collecting Flask-OpenTracing<1.2.0,>=1.1.0
  Downloading Flask-OpenTracing-1.1.0.tar.gz (8.2 kB)
Collecting opentracing<2.3.0,>=2.2.0
  Downloading opentracing-2.2.0.tar.gz (47 kB)
Collecting jaeger-client<4.2.0,>=4.1.0
  Downloading jaeger-client-4.1.0.tar.gz (80 kB)
Collecting grpcio-opentracing<1.2.0,>=1.1.4
  Downloading grpcio_opentracing-1.1.4-py3-none-any.whl (14 kB)
Collecting pyaml<20.0.0
  Downloading pyaml-19.12.0-py2.py3-none-any.whl (17 kB)
Collecting gunicorn<20.1.0,>=19.9.0
  Downloading gunicorn-20.0.4-py2.py3-none-any.whl (77 kB)
Collecting minio<6.0.0,>=4.0.9
  Downloading minio-5.0.8-py2.py3-none-any.whl (73 kB)
Collecting azure-storage-blob<3.0.0,>=2.0.1
  Downloading azure_storage_blob-2.1.0-py2.py3-none-any.whl (88 kB)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 17)) (45.2.0)
Collecting google-cloud-storage>=1.16.0
  Downloading google_cloud_storage-1.26.0-py2.py3-none-any.whl (75 kB)
Requirement already satisfied: prometheus_client<0.8.0,>=0.7.1 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 19)) (0.7.1)
Collecting black==19.10b0
  Downloading black-19.10b0-py36-none-any.whl (97 kB)
Collecting flake8==3.7.9
  Downloading flake8-3.7.9-py2.py3-none-any.whl (69 kB)
Collecting mypy<0.771
  Downloading mypy-0.770-cp36-cp36m-manylinux1_x86_64.whl (21.7 MB)
Requirement already satisfied: Pillow==7.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 5)) (7.0.0)
Collecting pytest==5.4.1
  Downloading pytest-5.4.1-py3-none-any.whl (246 kB)
Collecting pytest-cov==2.8.1
  Downloading pytest_cov-2.8.1-py2.py3-none-any.whl (18 kB)
Collecting tox<4.0.0
  Downloading tox-3.14.6-py2.py3-none-any.whl (81 kB)
Collecting coverage==4.5.4
  Downloading coverage-4.5.4-cp36-cp36m-manylinux1_x86_64.whl (205 kB)
Requirement already satisfied: pandas==1.0.1 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 14)) (1.0.1)
Collecting pip-licenses==2.1.1
  Downloading pip_licenses-2.1.1-py3-none-any.whl (13 kB)
Collecting click>=5.1
  Downloading click-7.1.1-py2.py3-none-any.whl (82 kB)
Collecting itsdangerous>=0.24
  Downloading itsdangerous-1.1.0-py2.py3-none-any.whl (16 kB)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (1.0.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (2.11.1)
Requirement already satisfied: Six in /usr/local/lib/python3.6/site-packages (from Flask-cors<4.0.0->-r requirements.txt (line 2)) (1.14.0)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (2019.11.28)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (2.9)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 4)) (1.25.8)
Collecting threadloop<2,>=1
  Downloading threadloop-1.0.2.tar.gz (4.9 kB)
Collecting thrift
  Downloading thrift-0.13.0.tar.gz (59 kB)
Collecting tornado<6,>=4.3
  Downloading tornado-5.1.1.tar.gz (516 kB)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->-r requirements.txt (line 13)) (5.3)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 15)) (2019.3)
Collecting configparser
  Downloading configparser-4.0.2-py2.py3-none-any.whl (22 kB)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 15)) (2.8.1)
Collecting azure-common>=1.1.5
  Downloading azure_common-1.1.25-py2.py3-none-any.whl (12 kB)
Collecting azure-storage-common~=2.1
  Downloading azure_storage_common-2.1.0-py2.py3-none-any.whl (47 kB)
Collecting google-cloud-core<2.0dev,>=1.2.0
  Downloading google_cloud_core-1.3.0-py2.py3-none-any.whl (26 kB)
Collecting google-auth<2.0dev,>=1.11.0
  Downloading google_auth-1.11.3-py2.py3-none-any.whl (76 kB)
Collecting google-resumable-media<0.6dev,>=0.5.0
  Downloading google_resumable_media-0.5.0-py2.py3-none-any.whl (38 kB)
Collecting regex
  Downloading regex-2020.2.20-cp36-cp36m-manylinux2010_x86_64.whl (690 kB)
Collecting pathspec<1,>=0.6
  Downloading pathspec-0.7.0-py2.py3-none-any.whl (25 kB)
Requirement already satisfied: attrs>=18.1.0 in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (19.3.0)
Collecting typed-ast>=1.4.0
  Downloading typed_ast-1.4.1-cp36-cp36m-manylinux1_x86_64.whl (737 kB)
Collecting toml>=0.9.4
  Downloading toml-0.10.0-py2.py3-none-any.whl (25 kB)
Collecting appdirs
  Downloading appdirs-1.4.3-py2.py3-none-any.whl (12 kB)
Collecting pycodestyle<2.6.0,>=2.5.0
  Downloading pycodestyle-2.5.0-py2.py3-none-any.whl (51 kB)
Collecting mccabe<0.7.0,>=0.6.0
  Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Collecting pyflakes<2.2.0,>=2.1.0
  Downloading pyflakes-2.1.1-py2.py3-none-any.whl (59 kB)
Requirement already satisfied: entrypoints<0.4.0,>=0.3.0 in /usr/local/lib/python3.6/site-packages (from flake8==3.7.9->-r requirements-dev.txt (line 3)) (0.3)
Collecting typing-extensions>=3.7.4
  Downloading typing_extensions-3.7.4.1-py3-none-any.whl (20 kB)
Collecting mypy-extensions<0.5.0,>=0.4.3
  Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Collecting pluggy<1.0,>=0.12
  Downloading pluggy-0.13.1-py2.py3-none-any.whl (18 kB)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.4.1->-r requirements-dev.txt (line 6)) (0.1.8)
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from pytest==5.4.1->-r requirements-dev.txt (line 6)) (1.5.0)
Collecting packaging
  Downloading packaging-20.3-py2.py3-none-any.whl (37 kB)
Collecting more-itertools>=4.0.0
  Downloading more_itertools-8.2.0-py3-none-any.whl (43 kB)
Collecting py>=1.5.0
  Downloading py-1.8.1-py2.py3-none-any.whl (83 kB)
Collecting filelock<4,>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0
  Downloading virtualenv-20.0.14-py2.py3-none-any.whl (4.6 MB)
Collecting PTable
  Downloading PTable-0.9.2.tar.gz (31 kB)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->-r requirements.txt (line 1)) (1.1.1)
Collecting cryptography
  Downloading cryptography-2.8-cp34-abi3-manylinux2010_x86_64.whl (2.3 MB)
Collecting google-api-core<2.0.0dev,>=1.16.0
  Downloading google_api_core-1.16.0-py2.py3-none-any.whl (70 kB)
Collecting cachetools<5.0,>=2.0.0
  Downloading cachetools-4.0.0-py3-none-any.whl (10 kB)
Collecting rsa<4.1,>=3.1.4
  Downloading rsa-4.0-py2.py3-none-any.whl (38 kB)
Collecting pyasn1-modules>=0.2.1
  Downloading pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->pytest==5.4.1->-r requirements-dev.txt (line 6)) (3.1.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->pytest==5.4.1->-r requirements-dev.txt (line 6)) (2.4.6)
Collecting importlib-resources<2,>=1.0; python_version < "3.7"
  Downloading importlib_resources-1.4.0-py2.py3-none-any.whl (20 kB)
Collecting distlib<1,>=0.3.0
  Downloading distlib-0.3.0.zip (571 kB)
Collecting cffi!=1.11.3,>=1.8
  Downloading cffi-1.14.0-cp36-cp36m-manylinux1_x86_64.whl (399 kB)
Collecting googleapis-common-protos<2.0dev,>=1.6.0
  Downloading googleapis-common-protos-1.51.0.tar.gz (35 kB)
Collecting pyasn1>=0.1.3
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
Building wheels for collected packages: Flask-OpenTracing, opentracing, jaeger-client, threadloop, thrift, tornado, PTable, distlib, googleapis-common-protos
  Building wheel for Flask-OpenTracing (setup.py): started
  Building wheel for Flask-OpenTracing (setup.py): finished with status 'done'
  Created wheel for Flask-OpenTracing: filename=Flask_OpenTracing-1.1.0-py3-none-any.whl size=9070 sha256=7912e126ac1c24fd6e80f6f9bd213ee55f576cc54366866d1b98d32244573d7e
  Stored in directory: /builder/home/.cache/pip/wheels/ad/4b/2d/24ff0da0a0b53c7c77ce59b843bcceaf644c88703241e59615
  Building wheel for opentracing (setup.py): started
  Building wheel for opentracing (setup.py): finished with status 'done'
  Created wheel for opentracing: filename=opentracing-2.2.0-py3-none-any.whl size=49319 sha256=3df31f3cb2e7b2f9199b3bb6dace9f55e399ce641259185d9ec69d41dfe518bf
  Stored in directory: /builder/home/.cache/pip/wheels/39/40/44/8bace79f4514e99786236c31f1df8d1b814ff02c1e08b1d697
  Building wheel for jaeger-client (setup.py): started
  Building wheel for jaeger-client (setup.py): finished with status 'done'
  Created wheel for jaeger-client: filename=jaeger_client-4.1.0-py3-none-any.whl size=64309 sha256=ae7a0098cb0c05dcd815a108701c29541a84378bd7a161887c8385b14297e2ad
  Stored in directory: /builder/home/.cache/pip/wheels/e9/9b/8c/503d0cc13b39a551c054515683ba1d15b40324c863dc442e66
  Building wheel for threadloop (setup.py): started
  Building wheel for threadloop (setup.py): finished with status 'done'
  Created wheel for threadloop: filename=threadloop-1.0.2-py3-none-any.whl size=3423 sha256=9dfce000f3ab9b868fe94e210395b88298145538fadba50938723ef881c843e1
  Stored in directory: /builder/home/.cache/pip/wheels/02/54/65/9f87de48fe8fcaaee30f279973d946ad55f9df56b93b3e78da
  Building wheel for thrift (setup.py): started
  Building wheel for thrift (setup.py): finished with status 'done'
  Created wheel for thrift: filename=thrift-0.13.0-cp36-cp36m-linux_x86_64.whl size=483897 sha256=bec2093a13d6d074591d1158ce29235c18a2afc66df9a2a149ba01ae65444fd5
  Stored in directory: /builder/home/.cache/pip/wheels/e0/38/fc/472fe18756b177b42096961f8bd3ff2dc5c5620ac399fce52d
  Building wheel for tornado (setup.py): started
  Building wheel for tornado (setup.py): finished with status 'done'
  Created wheel for tornado: filename=tornado-5.1.1-cp36-cp36m-linux_x86_64.whl size=462644 sha256=5a9f9506504a39b7a2c6b9e09ab4a0ca43103d8455aee96c4c7cade76f140c16
  Stored in directory: /builder/home/.cache/pip/wheels/64/74/71/e7a0d0eb4fc42cb20a17aec36f8f0b5b8c5e1ec19c701fd34a
  Building wheel for PTable (setup.py): started
  Building wheel for PTable (setup.py): finished with status 'done'
  Created wheel for PTable: filename=PTable-0.9.2-py3-none-any.whl size=22907 sha256=595456b55e677623942a2c19b56178d8c82473b1602bd72873ebcd429f22ef0c
  Stored in directory: /builder/home/.cache/pip/wheels/f3/65/67/71f473ec87ea4554d1bdfb1b5128cfe1414c2b113b72a1ee8e
  Building wheel for distlib (setup.py): started
  Building wheel for distlib (setup.py): finished with status 'done'
  Created wheel for distlib: filename=distlib-0.3.0-py3-none-any.whl size=340427 sha256=1a79ae2c5c73b83cff6d6d216579726723d10ac2cef4fee433261dd6f0f636dc
  Stored in directory: /builder/home/.cache/pip/wheels/33/d9/71/e4e3cac73529e1947df418af0f140cd7589d5d9ec0e17ecfc2
  Building wheel for googleapis-common-protos (setup.py): started
  Building wheel for googleapis-common-protos (setup.py): finished with status 'done'
  Created wheel for googleapis-common-protos: filename=googleapis_common_protos-1.51.0-py3-none-any.whl size=77592 sha256=e3f220cf64900362aeab892763a8f5804f4ffab1af05edfdfc1e4e6613c4f9dc
  Stored in directory: /builder/home/.cache/pip/wheels/35/8d/af/a922cb18800b31fadac3523cadf6c1efdf233b788fe7a4da70
Successfully built Flask-OpenTracing opentracing jaeger-client threadloop thrift tornado PTable distlib googleapis-common-protos
Installing collected packages: click, itsdangerous, Flask, Flask-cors, redis, flatbuffers, opentracing, Flask-OpenTracing, tornado, threadloop, thrift, jaeger-client, grpcio-opentracing, pyaml, gunicorn, configparser, minio, azure-common, pycparser, cffi, cryptography, azure-storage-common, azure-storage-blob, googleapis-common-protos, cachetools, pyasn1, rsa, pyasn1-modules, google-auth, google-api-core, google-cloud-core, google-resumable-media, google-cloud-storage, regex, pathspec, typed-ast, toml, appdirs, black, pycodestyle, mccabe, pyflakes, flake8, typing-extensions, mypy-extensions, mypy, pluggy, packaging, more-itertools, py, pytest, coverage, pytest-cov, filelock, importlib-resources, distlib, virtualenv, tox, PTable, pip-licenses, seldon-core
  Attempting uninstall: tornado
    Found existing installation: tornado 6.0.4
    Uninstalling tornado-6.0.4:
      Successfully uninstalled tornado-6.0.4
  Running setup.py develop for seldon-core
Successfully installed Flask-1.1.1 Flask-OpenTracing-1.1.0 Flask-cors-3.0.8 PTable-0.9.2 appdirs-1.4.3 azure-common-1.1.25 azure-storage-blob-2.1.0 azure-storage-common-2.1.0 black-19.10b0 cachetools-4.0.0 cffi-1.14.0 click-7.1.1 configparser-4.0.2 coverage-4.5.4 cryptography-2.8 distlib-0.3.0 filelock-3.0.12 flake8-3.7.9 flatbuffers-1.12 google-api-core-1.16.0 google-auth-1.11.3 google-cloud-core-1.3.0 google-cloud-storage-1.26.0 google-resumable-media-0.5.0 googleapis-common-protos-1.51.0 grpcio-opentracing-1.1.4 gunicorn-20.0.4 importlib-resources-1.4.0 itsdangerous-1.1.0 jaeger-client-4.1.0 mccabe-0.6.1 minio-5.0.8 more-itertools-8.2.0 mypy-0.770 mypy-extensions-0.4.3 opentracing-2.2.0 packaging-20.3 pathspec-0.7.0 pip-licenses-2.1.1 pluggy-0.13.1 py-1.8.1 pyaml-19.12.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pycodestyle-2.5.0 pycparser-2.20 pyflakes-2.1.1 pytest-5.4.1 pytest-cov-2.8.1 redis-3.4.1 regex-2020.2.20 rsa-4.0 seldon-core threadloop-1.0.2 thrift-0.13.0 toml-0.10.0 tornado-5.1.1 tox-3.14.6 typed-ast-1.4.1 typing-extensions-3.7.4.1 virtualenv-20.0.14
make: Leaving directory '/workspace/source/python'
cp ../../proto/prediction.proto ./proto
cd ../../proto/tensorflow && make create_protos
make[1]: Entering directory '/workspace/source/proto/tensorflow'
./create-protos.sh
Downloading proto files for master
make[1]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../../proto/tensorflow/tensorflow .
'../../proto/tensorflow/tensorflow' -> './tensorflow'
'../../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
cd ../../notebooks && make build_protos
make[1]: Entering directory '/workspace/source/notebooks'
cd ../proto/tensorflow && make create_protos
make[2]: Entering directory '/workspace/source/proto/tensorflow'
make[2]: Nothing to be done for 'create_protos'.
make[2]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../proto/tensorflow/tensorflow .
'../proto/tensorflow/tensorflow' -> './tensorflow'
'../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
cp ../proto/prediction.proto ./proto
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
make[1]: Leaving directory '/workspace/source/notebooks'
cd ../../proto/k8s && make create_protos
make[1]: Entering directory '/workspace/source/proto/k8s'
./create-k8s-protos.sh
Downloading proto files for master
Munging proto file packages
sed -i.bak 's|import "k8s.io/apiextensions-apiserver/|//import "k8s.io/apiextensions-apiserver/|' k8s.io/api/core/v1/generated.proto
rm k8s.io/api/core/v1/generated.proto.bak
make[1]: Leaving directory '/workspace/source/proto/k8s'
pip install -r dev_requirements.txt
Collecting pytest==5.3.1
  Downloading pytest-5.3.1-py3-none-any.whl (233 kB)
Collecting pytest-xdist==1.30.0
  Downloading pytest_xdist-1.30.0-py2.py3-none-any.whl (35 kB)
Requirement already satisfied: pytest-cov==2.8.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 3)) (2.8.1)
Collecting flaky==3.6.1
  Downloading flaky-3.6.1-py2.py3-none-any.whl (22 kB)
Collecting tenacity==6.0.0
  Downloading tenacity-6.0.0-py2.py3-none-any.whl (24 kB)
Requirement already satisfied: ipython==7.13.0 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 8)) (7.13.0)
Requirement already satisfied: nbconvert==5.6.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 9)) (5.6.1)
Requirement already satisfied: alibi==0.3.2 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 10)) (0.3.2)
Collecting matplotlib==3.1.3
  Downloading matplotlib-3.1.3-cp36-cp36m-manylinux1_x86_64.whl (13.1 MB)
Requirement already satisfied: tensorflow==1.15.2 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 12)) (1.15.2)
Requirement already satisfied: coverage==4.5.4 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 15)) (4.5.4)
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.5.0)
Requirement already satisfied: attrs>=17.4.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (19.3.0)
Requirement already satisfied: py>=1.5.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.8.1)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (20.3)
Requirement already satisfied: pluggy<1.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.13.1)
Requirement already satisfied: more-itertools>=4.0.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (8.2.0)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.1.8)
Collecting pytest-forked
  Downloading pytest_forked-1.1.3-py2.py3-none-any.whl (4.5 kB)
Requirement already satisfied: six in /usr/local/lib/python3.6/site-packages (from pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.14.0)
Collecting execnet>=1.1
  Downloading execnet-1.7.1-py2.py3-none-any.whl (39 kB)
Requirement already satisfied: decorator in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.4.2)
Requirement already satisfied: traitlets>=4.2 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.3.3)
Requirement already satisfied: setuptools>=18.5 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (45.2.0)
Requirement already satisfied: pickleshare in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.7.5)
Requirement already satisfied: jedi>=0.10 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.16.0)
Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (3.0.4)
Requirement already satisfied: pygments in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (2.5.2)
Requirement already satisfied: pexpect; sys_platform != "win32" in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (4.8.0)
Requirement already satisfied: backcall in /usr/local/lib/python3.6/site-packages (from ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.1.0)
Requirement already satisfied: entrypoints>=0.2.2 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.3)
Requirement already satisfied: jinja2>=2.4 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (2.11.1)
Requirement already satisfied: nbformat>=4.4 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (5.0.4)
Requirement already satisfied: mistune<2,>=0.8.1 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.8.4)
Requirement already satisfied: pandocfilters>=1.4.1 in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (1.4.2)
Requirement already satisfied: jupyter-core in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (4.6.3)
Requirement already satisfied: bleach in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (3.1.1)
Requirement already satisfied: testpath in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.4.4)
Requirement already satisfied: defusedxml in /usr/local/lib/python3.6/site-packages (from nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.6.0)
Requirement already satisfied: spacy in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.2.3)
Requirement already satisfied: numpy in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.18.1)
Requirement already satisfied: scikit-image in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.16.2)
Requirement already satisfied: requests in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.23.0)
Requirement already satisfied: scikit-learn in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.22.2.post1)
Requirement already satisfied: Pillow in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (7.0.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.1)
Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.6/site-packages (from alibi==0.3.2->-r dev_requirements.txt (line 10)) (4.8.2)
Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (1.1.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (2.4.6)
Requirement already satisfied: python-dateutil>=2.1 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (2.8.1)
Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.6/site-packages (from matplotlib==3.1.3->-r dev_requirements.txt (line 11)) (0.10.0)
Requirement already satisfied: tensorboard<1.16.0,>=1.15.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.15.0)
Requirement already satisfied: gast==0.2.2 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.2.2)
Requirement already satisfied: protobuf>=3.6.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.11.3)
Requirement already satisfied: keras-preprocessing>=1.0.5 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.1.0)
Requirement already satisfied: keras-applications>=1.0.8 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.0.8)
Requirement already satisfied: astor>=0.6.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.8.1)
Requirement already satisfied: wrapt>=1.11.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.12.0)
Requirement already satisfied: absl-py>=0.7.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.9.0)
Requirement already satisfied: wheel>=0.26; python_version >= "3" in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.34.2)
Requirement already satisfied: termcolor>=1.1.0 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.1.0)
Requirement already satisfied: tensorflow-estimator==1.15.1 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.15.1)
Requirement already satisfied: google-pasta>=0.1.6 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (0.1.8)
Requirement already satisfied: grpcio>=1.8.6 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.27.2)
Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.6/site-packages (from tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.2.0)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->pytest==5.3.1->-r dev_requirements.txt (line 1)) (3.1.0)
Collecting apipkg>=1.4
  Downloading apipkg-1.5-py2.py3-none-any.whl (4.9 kB)
Requirement already satisfied: ipython-genutils in /usr/local/lib/python3.6/site-packages (from traitlets>=4.2->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.2.0)
Requirement already satisfied: parso>=0.5.2 in /usr/local/lib/python3.6/site-packages (from jedi>=0.10->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.6.2)
Requirement already satisfied: ptyprocess>=0.5 in /usr/local/lib/python3.6/site-packages (from pexpect; sys_platform != "win32"->ipython==7.13.0->-r dev_requirements.txt (line 8)) (0.6.0)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from jinja2>=2.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (1.1.1)
Requirement already satisfied: jsonschema!=2.5.0,>=2.4 in /usr/local/lib/python3.6/site-packages (from nbformat>=4.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (3.2.0)
Requirement already satisfied: webencodings in /usr/local/lib/python3.6/site-packages (from bleach->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.5.1)
Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.2)
Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.6.0)
Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.0.3)
Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.4.1)
Requirement already satisfied: srsly<1.1.0,>=0.1.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.2)
Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.1.3)
Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (3.0.2)
Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.0.0)
Requirement already satisfied: thinc<7.4.0,>=7.3.0 in /usr/local/lib/python3.6/site-packages (from spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (7.3.1)
Requirement already satisfied: networkx>=2.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.4)
Requirement already satisfied: imageio>=2.3.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.8.0)
Requirement already satisfied: scipy>=0.19.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.4.1)
Requirement already satisfied: PyWavelets>=0.4.0 in /usr/local/lib/python3.6/site-packages (from scikit-image->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.1.1)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2019.11.28)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (1.25.8)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.9)
Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.6/site-packages (from scikit-learn->alibi==0.3.2->-r dev_requirements.txt (line 10)) (0.14.1)
Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.6/site-packages (from pandas->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2019.3)
Requirement already satisfied: soupsieve>=1.2 in /usr/local/lib/python3.6/site-packages (from beautifulsoup4->alibi==0.3.2->-r dev_requirements.txt (line 10)) (2.0)
Requirement already satisfied: werkzeug>=0.11.15 in /usr/local/lib/python3.6/site-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (1.0.0)
Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.6/site-packages (from tensorboard<1.16.0,>=1.15.0->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (3.2.1)
Requirement already satisfied: h5py in /usr/local/lib/python3.6/site-packages (from keras-applications>=1.0.8->tensorflow==1.15.2->-r dev_requirements.txt (line 12)) (2.10.0)
Requirement already satisfied: pyrsistent>=0.14.0 in /usr/local/lib/python3.6/site-packages (from jsonschema!=2.5.0,>=2.4->nbformat>=4.4->nbconvert==5.6.1->-r dev_requirements.txt (line 9)) (0.15.7)
Requirement already satisfied: tqdm<5.0.0,>=4.10.0 in /usr/local/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy->alibi==0.3.2->-r dev_requirements.txt (line 10)) (4.43.0)
Installing collected packages: pytest, pytest-forked, apipkg, execnet, pytest-xdist, flaky, tenacity, matplotlib
  Attempting uninstall: pytest
    Found existing installation: pytest 5.4.1
    Uninstalling pytest-5.4.1:
      Successfully uninstalled pytest-5.4.1
  Attempting uninstall: matplotlib
    Found existing installation: matplotlib 3.2.0
    Uninstalling matplotlib-3.2.0:
      Successfully uninstalled matplotlib-3.2.0
Successfully installed apipkg-1.5 execnet-1.7.1 flaky-3.6.1 matplotlib-3.1.3 pytest-5.3.1 pytest-forked-1.1.3 pytest-xdist-1.30.0 tenacity-6.0.0
pip install -e ../../python
Obtaining file:///workspace/source/python
Requirement already satisfied: Flask<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.1.1)
Requirement already satisfied: Flask-cors<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (3.0.8)
Requirement already satisfied: redis<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (3.4.1)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (2.23.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.18.1)
Requirement already satisfied: flatbuffers<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.12)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.27.2)
Requirement already satisfied: Flask-OpenTracing<1.2.0,>=1.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.1.0)
Requirement already satisfied: opentracing<2.3.0,>=2.2.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (2.2.0)
Requirement already satisfied: jaeger-client<4.2.0,>=4.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (4.1.0)
Requirement already satisfied: grpcio-opentracing<1.2.0,>=1.1.4 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (1.1.4)
Requirement already satisfied: pyaml<20.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (19.12.0)
Requirement already satisfied: gunicorn<20.1.0,>=19.9.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (20.0.4)
Requirement already satisfied: minio<6.0.0,>=4.0.9 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (5.0.8)
Requirement already satisfied: azure-storage-blob<3.0.0,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (2.1.0)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (45.2.0)
Requirement already satisfied: prometheus_client<0.8.0,>=0.7.1 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.2) (0.7.1)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.2) (2.11.1)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.2) (1.0.0)
Requirement already satisfied: click>=5.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.2) (7.1.1)
Requirement already satisfied: itsdangerous>=0.24 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.2) (1.1.0)
Requirement already satisfied: Six in /usr/local/lib/python3.6/site-packages (from Flask-cors<4.0.0->seldon-core==1.0.2) (1.14.0)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.2) (2019.11.28)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.2) (2.9)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.2) (1.25.8)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.2) (3.0.4)
Requirement already satisfied: threadloop<2,>=1 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.2) (1.0.2)
Requirement already satisfied: thrift in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.2) (0.13.0)
Requirement already satisfied: tornado<6,>=4.3 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.2) (5.1.1)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->seldon-core==1.0.2) (5.3)
Requirement already satisfied: configparser in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.2) (4.0.2)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.2) (2.8.1)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.2) (2019.3)
Requirement already satisfied: azure-common>=1.1.5 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.2) (1.1.25)
Requirement already satisfied: azure-storage-common~=2.1 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.2) (2.1.0)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->seldon-core==1.0.2) (1.1.1)
Requirement already satisfied: cryptography in /usr/local/lib/python3.6/site-packages (from azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.2) (2.8)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib/python3.6/site-packages (from cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.2) (1.14.0)
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.8->cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.2) (2.20)
Installing collected packages: seldon-core
  Attempting uninstall: seldon-core
    Found existing installation: seldon-core 1.0.2
    Uninstalling seldon-core-1.0.2:
      Successfully uninstalled seldon-core-1.0.2
  Running setup.py develop for seldon-core
Successfully installed seldon-core
# Run the core tests in parallel
pytest \
	--verbose \
	-s \
	-W ignore \
	-n "4" \
	-m "not sequential and not notebooks" 2>&1
============================= test session starts ==============================
platform linux -- Python 3.6.10, pytest-5.3.1, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /workspace/source/testing/scripts, inifile: setup.cfg
plugins: xdist-1.30.0, forked-1.1.3, flaky-3.6.1, cov-2.8.1
gw0 I / gw1 I / gw2 I / gw3 I
[gw0] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw1] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw2] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw3] linux Python 3.6.10 cwd: /workspace/source/testing/scripts
[gw0] Python 3.6.10 (default, Feb 28 2020, 10:20:25)  -- [GCC 8.3.0]
[gw2] Python 3.6.10 (default, Feb 28 2020, 10:20:25)  -- [GCC 8.3.0]
[gw3] Python 3.6.10 (default, Feb 28 2020, 10:20:25)  -- [GCC 8.3.0]
[gw1] Python 3.6.10 (default, Feb 28 2020, 10:20:25)  -- [GCC 8.3.0]
gw0 [27] / gw1 [27] / gw2 [27] / gw3 [27]

scheduling tests via LoadScheduling

test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] 
test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1] Error from server (SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "mymodel": Traffic must sum to 100 for multiple predictors]): error when creating "../resources/bad_duplicate_predictor_name.json": admission webhook "v1alpha2.vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "mymodel": Traffic must sum to 100 for multiple predictors]

[gw1] PASSED test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name 
test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model Error: uninstall: Release not loaded: mymab: release: not found

[gw1] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model Error: uninstall: Release not loaded: mymodel: release: not found

[gw3] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1] Error: uninstall: Release not loaded: mymodel: release: not found

[gw2] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] Error: uninstall: Release not loaded: mymodel: release: not found

[gw0] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] 
test_helm_charts_clusterwide.py::TestClusterWide::test_single_model 
test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch Error from server (SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit): error when creating "../resources/bad_name_mismatch.json": admission webhook "v1alpha2.vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit

[gw0] PASSED test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch 
test_prepackaged_servers.py::TestPrepack::test_sklearn 
test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model 
test_local_operators.py::TestLocalOperators::test_namespace_operator 
[gw1] PASSED test_local_operators.py::TestLocalOperators::test_namespace_operator 
test_prepackaged_servers.py::TestPrepack::test_mlflow Error from server (InternalError): error when creating "../../servers/mlflowserver/samples/elasticnet_wine.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/mlflowserver/samples/elasticnet_wine.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/mlflowserver/samples/elasticnet_wine.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/mlflowserver/samples/elasticnet_wine.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/mlflowserver/samples/elasticnet_wine.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

[gw0] FAILED test_prepackaged_servers.py::TestPrepack::test_sklearn 
[gw3] FAILED test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model 
test_prepackaged_servers.py::TestPrepack::test_tfserving Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

test_prepackaged_servers.py::TestPrepack::test_xgboost Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

[gw2] FAILED test_helm_charts_clusterwide.py::TestClusterWide::test_single_model Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/xgboostserver/samples/iris.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../../servers/tfserving/samples/mnist_rest.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

test_local_operators.py::TestLocalOperators::test_labelled_operator Error from server (InternalError): error when creating "../resources/model_controller_id.yaml": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.test-labelled-operator.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.108.228.189:443: connect: connection refused

[gw2] PASSED test_local_operators.py::TestLocalOperators::test_labelled_operator 
[gw1] FAILED test_prepackaged_servers.py::TestPrepack::test_mlflow 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-ambas] Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-istio] Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused
Error from server (InternalError): error when creating "../resources/graph1.json": Internal error occurred: failed calling webhook "v1alpha2.mseldondeployment.kb.io": Post https://seldon-webhook-service.seldon-system.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.98.65.6:443: connect: connection refused

[gw3] FAILED test_prepackaged_servers.py::TestPrepack::test_xgboost 
[gw0] FAILED test_prepackaged_servers.py::TestPrepack::test_tfserving 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-istio] 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-istio] 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-istio] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph3.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-ambas] 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-istio] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-ambas] 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph5.json-True-istio] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-istio] 
[gw1] FAILED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph2.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-ambas] 
[gw3] PASSED test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-ambas] 
test_tracing.py::test_tracing_rest 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-istio] 
[gw3] PASSED test_tracing.py::test_tracing_rest 
[gw0] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph8.json-True-ambas] 
test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-istio] 
[gw1] PASSED test_rolling_updates.py::test_rolling_deployment[graph1.json-graph4.json-True-ambas] 
[gw2] PASSED test_rolling_updates.py::test_rolling_deployment[graph7.json-graph8.json-False-istio] 

=================================== FAILURES ===================================
___________________________ TestPrepack.test_sklearn ___________________________
[gw0] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7f6ddbd84630>
namespace = 'test-sklearn'

    def test_sklearn(self, namespace):
        spec = "../../servers/sklearnserver/samples/iris.yaml"
        retry_run(f"kubectl apply -f {spec} -n {namespace}")
        wait_for_status("sklearn", namespace)
>       wait_for_rollout("sklearn", namespace)

test_prepackaged_servers.py:20: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:118: in wait_for_rollout
    deployment_names = get_deployment_names(sdep_name, namespace)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'sklearn', namespace = 'test-sklearn', attempts = 20, sleep = 5

    def get_deployment_names(sdep_name, namespace, attempts=20, sleep=5):
        for _ in range(attempts):
            ret = run(
                f"kubectl get -n {namespace} sdep {sdep_name} -o json",
                shell=True,
                stdout=subprocess.PIPE,
            )
            if ret.returncode == 0:
                logging.info(f"Successfully waited for SeldonDeployment {sdep_name}")
                break
            logging.warning(
                f"Unsuccessful wait command but retrying for SeldonDeployment {sdep_name}"
            )
            time.sleep(sleep)
        assert ret.returncode == 0, "Failed to get deployment names: non-zero return code"
        data = json.loads(ret.stdout)
        # The `deploymentStatus` is dictionary which keys are names of deployments
>       deployment_names = list(data["status"]["deploymentStatus"])
E       KeyError: 'status'

seldon_e2e_utils.py:105: KeyError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
______________________ TestClusterWide.test_abtest_model _______________________
[gw3] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_helm_charts_clusterwide.TestClusterWide object at 0x7ff03d227eb8>
namespace = 'test-abtest-model'

    def test_abtest_model(self, namespace):
        run(
            f"helm install myabtest ../../helm-charts/seldon-abtest --namespace {namespace}",
            shell=True,
            check=True,
        )
        wait_for_status("myabtest", namespace)
>       wait_for_rollout("myabtest", namespace, expected_deployments=2)

test_helm_charts_clusterwide.py:42: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:118: in wait_for_rollout
    deployment_names = get_deployment_names(sdep_name, namespace)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'myabtest', namespace = 'test-abtest-model', attempts = 20
sleep = 5

    def get_deployment_names(sdep_name, namespace, attempts=20, sleep=5):
        for _ in range(attempts):
            ret = run(
                f"kubectl get -n {namespace} sdep {sdep_name} -o json",
                shell=True,
                stdout=subprocess.PIPE,
            )
            if ret.returncode == 0:
                logging.info(f"Successfully waited for SeldonDeployment {sdep_name}")
                break
            logging.warning(
                f"Unsuccessful wait command but retrying for SeldonDeployment {sdep_name}"
            )
            time.sleep(sleep)
        assert ret.returncode == 0, "Failed to get deployment names: non-zero return code"
        data = json.loads(ret.stdout)
        # The `deploymentStatus` is dictionary which keys are names of deployments
>       deployment_names = list(data["status"]["deploymentStatus"])
E       KeyError: 'status'

seldon_e2e_utils.py:105: KeyError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
______________________ TestClusterWide.test_single_model _______________________
[gw2] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_helm_charts_clusterwide.TestClusterWide object at 0x7f7c962e6ef0>
namespace = 'test-single-model'

    def test_single_model(self, namespace):
        run(
            f"helm install mymodel ../../helm-charts/seldon-single-model --namespace {namespace}",
            shell=True,
            check=True,
        )
        wait_for_status("mymodel", namespace)
        wait_for_rollout("mymodel", namespace)
        initial_rest_request("mymodel", namespace)
        logging.warning("Test Ambassador REST gateway")
        r = rest_request_ambassador("mymodel", namespace, API_AMBASSADOR)
>       logging.warning(r.json())

test_helm_charts_clusterwide.py:29: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.6/site-packages/requests/models.py:898: in json
    return complexjson.loads(self.text, **kwargs)
/usr/local/lib/python3.6/json/__init__.py:354: in loads
    return _default_decoder.decode(s)
/usr/local/lib/python3.6/json/decoder.py:339: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <json.decoder.JSONDecoder object at 0x7f7ce1181278>, s = '', idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` beginning with
        a JSON document) and return a 2-tuple of the Python
        representation and the index in ``s`` where the document ended.
    
        This can be used to decode a JSON document from a string that may
        have extraneous data at the end.
    
        """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration as err:
>           raise JSONDecodeError("Expecting value", s, err.value) from None
E           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

/usr/local/lib/python3.6/json/decoder.py:357: JSONDecodeError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:212 Bad status:404
WARNING  root:seldon_e2e_utils.py:212 Bad status:404
WARNING  root:seldon_e2e_utils.py:212 Bad status:404
WARNING  root:seldon_e2e_utils.py:212 Bad status:404
WARNING  root:test_helm_charts_clusterwide.py:27 Test Ambassador REST gateway
___________________________ TestPrepack.test_mlflow ____________________________
[gw1] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7f062500c9e8>
namespace = 'test-mlflow'

    def test_mlflow(self, namespace):
        spec = "../../servers/mlflowserver/samples/elasticnet_wine.yaml"
        retry_run(f"kubectl apply -f {spec} -n {namespace}")
        wait_for_status("mlflow", namespace)
>       wait_for_rollout("mlflow", namespace)

test_prepackaged_servers.py:68: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:118: in wait_for_rollout
    deployment_names = get_deployment_names(sdep_name, namespace)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'mlflow', namespace = 'test-mlflow', attempts = 20, sleep = 5

    def get_deployment_names(sdep_name, namespace, attempts=20, sleep=5):
        for _ in range(attempts):
            ret = run(
                f"kubectl get -n {namespace} sdep {sdep_name} -o json",
                shell=True,
                stdout=subprocess.PIPE,
            )
            if ret.returncode == 0:
                logging.info(f"Successfully waited for SeldonDeployment {sdep_name}")
                break
            logging.warning(
                f"Unsuccessful wait command but retrying for SeldonDeployment {sdep_name}"
            )
            time.sleep(sleep)
        assert ret.returncode == 0, "Failed to get deployment names: non-zero return code"
        data = json.loads(ret.stdout)
        # The `deploymentStatus` is dictionary which keys are names of deployments
>       deployment_names = list(data["status"]["deploymentStatus"])
E       KeyError: 'status'

seldon_e2e_utils.py:105: KeyError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/mlflowserver/samples/elasticnet_wine.yaml -n test-mlflow
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/mlflowserver/samples/elasticnet_wine.yaml -n test-mlflow
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/mlflowserver/samples/elasticnet_wine.yaml -n test-mlflow
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/mlflowserver/samples/elasticnet_wine.yaml -n test-mlflow
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/mlflowserver/samples/elasticnet_wine.yaml -n test-mlflow
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
___________________________ TestPrepack.test_xgboost ___________________________
[gw3] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7ff03d107198>
namespace = 'test-xgboost'

    def test_xgboost(self, namespace):
        spec = "../../servers/xgboostserver/samples/iris.yaml"
        retry_run(f"kubectl apply -f {spec}  -n {namespace}")
        wait_for_status("xgboost", namespace)
>       wait_for_rollout("xgboost", namespace)

test_prepackaged_servers.py:53: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:118: in wait_for_rollout
    deployment_names = get_deployment_names(sdep_name, namespace)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'xgboost', namespace = 'test-xgboost', attempts = 20, sleep = 5

    def get_deployment_names(sdep_name, namespace, attempts=20, sleep=5):
        for _ in range(attempts):
            ret = run(
                f"kubectl get -n {namespace} sdep {sdep_name} -o json",
                shell=True,
                stdout=subprocess.PIPE,
            )
            if ret.returncode == 0:
                logging.info(f"Successfully waited for SeldonDeployment {sdep_name}")
                break
            logging.warning(
                f"Unsuccessful wait command but retrying for SeldonDeployment {sdep_name}"
            )
            time.sleep(sleep)
        assert ret.returncode == 0, "Failed to get deployment names: non-zero return code"
        data = json.loads(ret.stdout)
        # The `deploymentStatus` is dictionary which keys are names of deployments
>       deployment_names = list(data["status"]["deploymentStatus"])
E       KeyError: 'status'

seldon_e2e_utils.py:105: KeyError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/xgboostserver/samples/iris.yaml  -n test-xgboost
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
__________________________ TestPrepack.test_tfserving __________________________
[gw0] linux -- Python 3.6.10 /usr/local/bin/python

self = <test_prepackaged_servers.TestPrepack object at 0x7f6ddbd84400>
namespace = 'test-tfserving'

    def test_tfserving(self, namespace):
        spec = "../../servers/tfserving/samples/mnist_rest.yaml"
        retry_run(f"kubectl apply -f {spec}  -n {namespace}")
        wait_for_status("tfserving", namespace)
>       wait_for_rollout("tfserving", namespace)

test_prepackaged_servers.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:118: in wait_for_rollout
    deployment_names = get_deployment_names(sdep_name, namespace)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'tfserving', namespace = 'test-tfserving', attempts = 20, sleep = 5

    def get_deployment_names(sdep_name, namespace, attempts=20, sleep=5):
        for _ in range(attempts):
            ret = run(
                f"kubectl get -n {namespace} sdep {sdep_name} -o json",
                shell=True,
                stdout=subprocess.PIPE,
            )
            if ret.returncode == 0:
                logging.info(f"Successfully waited for SeldonDeployment {sdep_name}")
                break
            logging.warning(
                f"Unsuccessful wait command but retrying for SeldonDeployment {sdep_name}"
            )
            time.sleep(sleep)
        assert ret.returncode == 0, "Failed to get deployment names: non-zero return code"
        data = json.loads(ret.stdout)
        # The `deploymentStatus` is dictionary which keys are names of deployments
>       deployment_names = list(data["status"]["deploymentStatus"])
E       KeyError: 'status'

seldon_e2e_utils.py:105: KeyError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../../servers/tfserving/samples/mnist_rest.yaml  -n test-tfserving
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
_________ test_rolling_deployment[graph1.json-graph2.json-True-ambas] __________
[gw1] linux -- Python 3.6.10 /usr/local/bin/python

namespace = 'test-rolling-deployment-graph1-json-graph2-json-true-ambas'
api_gateway = 'localhost:8003', from_deployment = 'graph1.json'
to_deployment = 'graph2.json', change = True

    @pytest.mark.flaky(max_runs=2)
    @with_api_gateways
    @pytest.mark.parametrize(
        "from_deployment,to_deployment,change",
        [
            ("graph1.json", "graph2.json", True),  # New image version
            (
                "graph1.json",
                "graph3.json",
                True,
            ),  # New image version and new name of container
            ("graph1.json", "graph4.json", True),  # New resource request but same image
            ("graph1.json", "graph5.json", True),  # Update with multi-deployment new model
            ("graph1.json", "graph8.json", True),  # From v1alpha2 to v1
            ("graph7.json", "graph8.json", False),  # From v1alpha3 to v1
        ],
    )
    def test_rolling_deployment(
        namespace, api_gateway, from_deployment, to_deployment, change
    ):
        if api_gateway == API_ISTIO_GATEWAY:
            retry_run(f"kubectl create -f ../resources/seldon-gateway.yaml -n {namespace}")
    
        from_file_path = to_resources_path(from_deployment)
        retry_run(f"kubectl apply -f {from_file_path} -n {namespace}")
        wait_for_status("mymodel", namespace)
        wait_for_rollout("mymodel", namespace)
        assert_model("mymodel", namespace, initial=True, endpoint=api_gateway)
    
        old_pod_name = get_pod_name_for_sdep("mymodel", namespace)[0]
        to_file_path = to_resources_path(to_deployment)
    
        def _update_model():
            retry_run(f"kubectl apply -f {to_file_path} -n {namespace}")
            if change:
                wait_for_pod_shutdown(old_pod_name, namespace)
            wait_for_status("mymodel", namespace)
            time.sleep(2)  # Wait a little after deployment marked Available
    
>       assert_model_during_op(_update_model, "mymodel", namespace, endpoint=api_gateway)

test_rolling_updates.py:269: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
seldon_e2e_utils.py:502: in assert_model_during_op
    raise err
seldon_e2e_utils.py:494: in assert_model_during_op
    assert_model(*assert_args, **assert_kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sdep_name = 'mymodel'
namespace = 'test-rolling-deployment-graph1-json-graph2-json-true-ambas'
initial = False, endpoint = 'localhost:8003'

    def assert_model(sdep_name, namespace, initial=False, endpoint=API_AMBASSADOR):
        _request = initial_rest_request if initial else rest_request
        r = _request(sdep_name, namespace, endpoint=endpoint)
    
>       assert r is not None
E       AssertionError

seldon_e2e_utils.py:512: AssertionError
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:154 Unsuccessful command but retrying: kubectl apply -f ../resources/graph1.json -n test-rolling-deployment-graph1-json-graph2-json-true-ambas
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:212 Bad status:503
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:172 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:212 Bad status:503
===Flaky Test Report===

test_rolling_deployment[graph1.json-graph3.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph5.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph7.json-graph8.json-False-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph2.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph5.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph8.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph2.json-True-ambas] failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	
	[<TracebackEntry /workspace/source/testing/scripts/test_rolling_updates.py:269>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:502>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:494>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:512>]
test_rolling_deployment[graph1.json-graph2.json-True-ambas] failed; it passed 0 out of the required 1 times.
	<class 'AssertionError'>
	
	[<TracebackEntry /workspace/source/testing/scripts/test_rolling_updates.py:269>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:502>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:494>, <TracebackEntry /workspace/source/testing/scripts/seldon_e2e_utils.py:512>]
test_rolling_deployment[graph1.json-graph4.json-True-ambas] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph3.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph4.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph1.json-graph8.json-True-istio] passed 1 out of the required 1 times. Success!
test_rolling_deployment[graph7.json-graph8.json-False-istio] passed 1 out of the required 1 times. Success!

===End Flaky Test Report===
=================== 7 failed, 20 passed in 807.38s (0:13:27) ===================
make: *** [Makefile:105: test_parallel] Error 1
Test returned errors
kind delete cluster
Deleting cluster "kind" ...
Stopping Docker: dockerProgram process in pidfile '/var/run/docker-ssd.pid', 1 process(es), refused to die.
[31m
Pipeline failed on stage 'end-to-end' : container 'step-test-end-to-end'. The execution of the pipeline has stopped.[0m
